api/views/admin/admin.py

```diff
@@ -563,7 +570,9 @@ def get_sql_rows(table):
                     'first-name': user.FirstName,
                     'last-name': user.LastName,
                     'access-rights-sdo': user.AccessRightsSdo,
-                    'access-rights-vendor': user.AccessRightsVendor}
+                    'access-rights-vendor': user.AccessRightsVendor,
+                    'motivation': user.Motivation,
+                    'registeration-datetime': user.RegisterationDatetime}
```

Comment:
There is a type: **user.RegisterationDatetime** -> **user.RegistrationDatetime**